gubernator/templates/build.html

```diff
@@ -45,6 +45,11 @@
 			<tr><td>{{k}}<td>{{v}}
 			% endfor
 		% endif
+		% if finished and 'metadata' in finished
+			% for k, v in finished['metadata']|dictsort
+			<tr><td>{{k}}<td>{{v}}
```

Comment:
Yeah! This thread looks just like one Ryan had with me a few months ago :smirk_cat: